### PR TITLE
Support preselected items

### PR DESCRIPTION
**Pull request by kf99916**
The feature lets users can select/deselect items back and forth.

### CODE REVIEW

File: YPImagePickerExample/YPImagePickerExample/ExampleViewController.swift [hunk 1]
```diff
@@ -169,7 +169,7 @@ class ExampleViewController: UIViewController {
         //
         //config.library.options = options
 
-        let picker = YPImagePicker(configuration: config)
+        let picker = YPImagePicker(configuration: config, items: selectedItems)
```
Comment (NikKovIos) [inline YPImagePickerExample/YPImagePickerExample/ExampleViewController.swift]: Sorry, but this is a major update for API. Please use YPConfigLibrary to put this configuration into.

File: Source/YPPickerVC.swift [hunk 2]
```diff
@@ -57,7 +59,7 @@ open class YPPickerVC: YPBottomPager, YPBottomPagerDelegate {
         
         // Library
         if YPConfig.screens.contains(.library) {
-            libraryVC = YPLibraryVC()
+            libraryVC = YPLibraryVC(items: preSelectedItems)
```
Comment (NikKovIos) [inline Source/YPPickerVC.swift]: There and in other place, not need to camel case for `preselected`.

File: Source/Pages/Gallery/YPLibraryVC+CollectionView.swift [hunk 3]
```diff
@@ -139,6 +139,10 @@ extension YPLibraryVC: UICollectionViewDelegate {
         
         // Set correct selection number
         if let index = selection.firstIndex(where: { $0.assetIdentifier == asset.localIdentifier }) {
+            let currentSelection = selection[index]
```
Comment (NikKovIos) [inline Source/Pages/Gallery/YPLibraryVC+CollectionView.swift:142]: Please add a comment in code what is happening there.

**Review by NikKovIos — CHANGES_REQUESTED** [review]

**Comment by kf99916** [comment]
@NikKovIos , the PR is refined according to your suggestion. Thanks you.

**Comment by NikKovIos** [comment]
@kf99916 Mark as resolved appropriate conversations

**Comment by touyu** [comment]
When will this be merged? I want this feature.

**Pull request merged**